### ISSUE TITLE
Improve accessibility of the `[clone]` shortcode

### ISSUE THREAD

**Issue by bobbingwide** [bobbingwide/oik-clone#76]
When WAVE is run against the output of the `[clone]` shortcode it produces a **Missing form label** error for each checkbox associated with a post to be cloned.

There are two errors for this form.

```
<form method="post">Posts :2
<br>
<a href="https://s.b/oikcom/shortcode_example/bw_logo-link/">11129</a>
 11129 0 self shortcode_example 1 / 2 
<input type="hidden" name="clone[11129]" value="0">
<input type="checkbox" name="clone[11129]" id="clone[11129]">
<br>
<a href="https://s.b/oikcom/oik_shortcodes/bw_logo/">5894</a>
 11129 1 related oik_shortcodes 2 / 2 
<input type="hidden" name="clone[5894]" value="0">
<input type="checkbox" name="clone[5894]" id="clone[5894]">
<input type="hidden" id="_oik_clone_form11129" name="_oik_clone_form11129" value="fb9fd7df25">
<br>
<input type="submit" name="_oik_clone_submit_11129" value="Clone" fdprocessedid="g0hboa">
</form>
```

**Comment by bobbingwide** [comment]
WAVE says this...
### What To Do

If a text label for a form control is visible, use the <label> element to associate it with its respective form control. If there is no visible label, either provide an associated label, add a descriptive title attribute to the form control, or reference the label(s) using aria-labelledby. Labels are not required for image, submit, reset, button, or hidden form controls.

### The Algorithm... in English

An <input> (except types of image, submit, reset, button, or hidden), <select>, or <textarea> does not have a properly associated label. 

A properly associated label is:
- a non-hidden <label> element with a for attribute value that is equal to the id of a unique form control
- a <label> element that surrounds the form control, does not surround any other form controls, and does not reference another element with its for attribute
- a non-empty title attribute, or
- a non-empty aria-labelledby attribute
